tests/unittests/test_datasource/test_ec2.py

```diff
@@ -658,9 +811,10 @@ def test_convert_ec2_metadata_network_config_handles_local_dhcp4(self):
             network_metadata_ipv6['interfaces']['macs'][self.mac1])
         nic1_metadata['local-ipv4s'] = '172.3.3.15'
         nic1_metadata.pop('public-ipv4s')
-        expected = {'version': 1, 'config': [
-            {'mac_address': self.mac1, 'type': 'physical',
-             'name': 'eth9', 'subnets': [{'type': 'dhcp4'}]}]}
+        expected = {'version': 2, 'ethernets': {'eth9': {
+            'match': {'macaddress': self.mac1.lower()}, 'set-name': 'eth9',
```

Comment:
drop mac1.lower()

Reply:
dropped with the expection of one test  test_network_config_property_set_dhcp4_on_private_ipv4.
That test  needs the lower call because the input to the function under test is the response from m_get_interfaces_by_mac which contains uppercase mac addresses from the system.